src/algebra/ordered_group.lean

```diff
@@ -1019,3 +1215,58 @@ instance [linear_ordered_comm_group α] : linear_ordered_add_comm_group (additiv
   ..additive.ordered_add_comm_group }
 
 end type_tags
+
+section norm_num_lemmas
+/- The following lemmas are stated so that the `norm_num` tactic can use them with the
+expected signatures.  -/
```

Comment:
maybe move this to `tactic.norm_num` and `tactic.linarith.lemmas`?

Reply:
Although I am not sure, I think that some of the lemmas that I had restated to fix `norm_num` are also used by `linarith`.  Thus, moving the lemmas to `norm_num` breaks `linarith`.

A good solution to this would probably be to create a file with `norm_num + linarith` lemmas and import that file from both `norm_num` and `linarith`.  However, I may not do this in this PR: I am tempted to simply leave these lemmas where they are now.

Does this seem reasonable?

Reply:
Sounds good!